www/js/submit.js

```diff
@@ -0,0 +1,17 @@
+
+$(document).ready(function(){
+  var $submitButton = $(".submit-button");
+  $submitButton.click(function() {
+     var ask = window.confirm("You are offline right now, ready to store it?");
+         if (ask) {
+                window.alert("This post was successfully stored.");
+                    
+                window.location = "homepage.html";
+                    
```

Comment:
Since maybe there has unsubmitted file pending. This action just to make a demo to simulate....  Does it matter or not?